arangod/Aql/TraversalExecutor.cpp

```diff
@@ -194,7 +194,10 @@ TraversalExecutorInfos::traversalEnumerator() const {
 }
```

Comment:
The code above this does have exactly the same issue!
We call both of the Methods next to each other, but only one actually has a pointer (depending on the "isRefactor" state)
So I think to make this correct, all changes in the PR need to be applied for the REFACTOR _traversalEnumerator as well.